ansible/test_inventory

```diff
@@ -0,0 +1,5 @@
+[master]
```

Comment:
This file shouldn't be updated - might have become unsynced with master

Reply:
Seems like in your branch, there is no test_inventory anymore, should I just remove this file?

Reply:
I use the file locally for testing, and I think it may have been checked in by accident so I removed it.